api/models/project.go

```diff
@@ -0,0 +1,177 @@
+package models
```

Comment:
@Dotunj Since we aim to document and manage the public API separately from the dashboard API. This file should be in `api/public/models.go`